apps/site/util/url.ts

```diff
@@ -17,73 +17,91 @@ export const getNodeApiUrl = (version: string) => {
     : `${DIST_URL}${version}/docs/api/`;
 };
 
-export const getNodeDownloadUrl = (
-  versionWithPrefix: string,
-  os: OperatingSystem | 'LOADING',
-  platform: Platform = 'x64',
-  kind: DownloadKind = 'installer'
-) => {
-  const baseURL = `${DIST_URL}${versionWithPrefix}`;
+type DownloadOptions = {
+  /** The Node.js version string, must include the 'v' prefix (e.g., 'v20.12.2'). */
+  version: string;
+  /** The target operating system. Defaults to 'LOADING'. */
+  os?: OperatingSystem | 'LOADING';
+  /** The target platform/architecture (e.g., 'x64', 'arm64'). Defaults to 'x64'. */
+  platform?: Platform;
+  /** The type of download artifact. Can be 'installer', 'binary', 'source', or 'shasum'. Defaults to 'installer'. */
+  kind?: DownloadKind;
+};
+
+/**
+ * Generates a Node.js download URL for the given options
+ */
+export const getNodeDownloadUrl = ({
+  version,
```

Comment:
But this should be the version withPrefix, no? Can this be clarified? Might make sense to just keep the variable name as versionWithPrefix :)